cool-core/src/main/java/com/nus/cool/core/cohort/filter/AllFilter.java

```diff
@@ -0,0 +1,96 @@
+package com.nus.cool.core.cohort.filter;
+
+import com.nus.cool.core.cohort.storage.Scope;
+import com.nus.cool.core.io.readstore.MetaChunkRS;
+import java.util.BitSet;
+import java.util.List;
+
+/**
+ * All filter class.
+ */
+public class AllFilter implements Filter {
+
+  private static final FilterType type = FilterType.ALL;
+
+
+  /**
+   * Accept all values.
+   *
+   * @param value the Input Value to check
+   * @return true
+   */
+  @Override
+  public Boolean accept(Integer value) {
+    return true;
+  }
+
+  /**
+   *  Accept all values.
+   *
+   * @param values the batch of input value to check
+   * @return null
+   */
+  @Override
+  public BitSet accept(List<Integer> values) {
+    return null;
+  }
+
+  /**
+   *  Accept all values.
+   *
+   * @param values Scope of time_min-time_max
+   * @return true
+   */
+  @Override
+  public boolean accept(Scope values) {
+    return true;
+  }
+
+  /**
+   *  Accept all values.
+   *
+   * @param value value
+   * @return true
+   */
+  @Override
+  public Boolean accept(String value) {
+    return true;
+  }
+
+  /**
+   *  Accept all values.
+   *
+   * @param values values
+   * @return null
+   */
+  @Override
+  public BitSet accept(String[] values) {
```

Comment:
This class is added by @KimballCai . From the doc in file, I' m not sure the roles of this class.

Reply:
I don't see the need to add this ALL Filter. 
Personally, if you want to accept all input value, you can set the filter None, and leave this logic into other place.
I don't like it.

Reply:
once the cohortSelect item is null, we need a filter to support your logic. AllFilter is a temporary solution, and we may improve it and update the logic later.

Reply:
In my opinion, it is ok. Having this filter means that we do not need to handle "if the filter is null" in all other places, hence making those codes cleaner. The alternative may be using java Optional instead of passing null around.